storm-core/test/clj/backtype/storm/supervisor_test.clj

```diff
@@ -261,8 +260,8 @@
                                 (str "-Dstorm.id=" mock-storm-id)
                                 (str "-Dworker.id=" mock-worker-id)
                                 (str "-Dworker.port=" mock-port)
-                               "-Dstorm.log.dir=/logs"
-                               "-Dlog4j.configurationFile=/log4j2/worker.xml"
+                                (str "-Dstorm.log.dir=" file-path-separator "logs")
+                                (str "-Dlog4j.configurationFile=" file-path-separator "log4j2" file-path-separator "worker.xml")
```

Comment:
@HeartSaVioR  sorry, I didn't see STORM-1463 before.  I think we should find whole of things and fix it.  Thank you @HeartSaVioR